niceman/distributions/tests/test_conda.py

```diff
@@ -0,0 +1,70 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil; coding: utf-8 -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the niceman package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+import collections
+import os
+
+import sys
+from appdirs import AppDirs
+from subprocess import call
+
+import yaml
+import attr
+from niceman.formats.niceman import NicemanProvenance
+from niceman.tests.utils import skip_if_no_network
+
+try:
+    import apt
+except ImportError:
+    apt = None
+
+import json
+
+from niceman.distributions.conda import CondaTracer
+
+
+def create_test_conda(test_dir):
+    if os.path.exists(test_dir):
+        return
+    call("mkdir -p " + test_dir + "; "
+         "cd " + test_dir + "; "
+         "curl -O https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh; "
+         "bash -b Miniconda2-latest-Linux-x86_64.sh -b -p ./miniconda; "
```

Comment:
it should be real easy to generalize it to at least OSX right?

Reply:
I tried to address OSX miniconda testing in c3fe30c .  Once OSX under travis is up, then I get to test it :)